packages/office-ui-fabric-react/src/components/Icon/examples/Icon.SvgFactory.Example.tsx

```diff
@@ -4,21 +4,17 @@ import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
 import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
 
 const classes = mergeStyleSets({
-  root: {
-    selectors: {
-      '> *': {
-        height: 50,
-        width: 50,
-        marginRight: 25,
-      },
-    },
-  },
   cell: {
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'center',
     margin: '80px',
     float: 'left',
+    height: '50px',
+    width: '50px',
+  },
```

Comment:
1. remove merge-styles dependency from package.json
2. Add load-themed-styles because it is (currently) used when we import scss files.

Reply:
this is example code.